java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java

```diff
@@ -339,7 +339,7 @@ private String buildWorkerCommandRaylet() {
 
   private void startObjectStore() {
     try (FileUtil.TempFile plasmaStoreFile = FileUtil
-        .getTempFileFromResource("external/plasma/plasma_store_server")) {
+        .getTempFileFromResource("plasma_store_server")) {
```

Comment:
Why change this?

Reply:
`plasma_store_server ` binary is copied to `native_dependencies` directory, then packaged into jar root directory